charts/portworx/values.yaml

```diff
@@ -13,20 +13,20 @@ etcdEndPoint:                         # The ETCD endpoint. Should be in the form
 clusterName: mycluster                # This is the default. please change it to your cluster name.
 usefileSystemDrive: false             # true/false Instructs PX to use an unmounted Drive even if it has a filesystem.
 usedrivesAndPartitions: false         # Defaults to false. Change to true and PX will use unmounted drives and partitions.
-secretType: none                      # Defaults to None, but can be kvdb/k8s/aws-kms/vault/ibm-kp
+secretType: k8s                      # Defaults to None, but can be kvdb/k8s/aws-kms/vault/ibm-kp
```

Comment:
NOTE: Comment is incorrect (Default is no longer none).

Please also ensure we are keeping the default identical to the defaults on http://install.portworx.com

Reply:
Updated the comments